### DISCUSSION TITLE
dag_file_processor_timeout behaviour

### DISCUSSION THREAD

**Discussion by ErandM50** [apache/airflow#40031] [Q&A]
From my understanding **dag_file_processor_timeout** is relevant for each dag file processing. So if processing each dag individually exceeds **dag_file_processor_timeout**, the processor times out. However, I have noticed that it seems like it is the time it takes to run through a whole loop of **DagFileProcessorManager** where it processes all dag files. From the dag report my largest import time is 30 sec, but i need to have the timeout set to over 3 min (which is the time it takes to do _airflow dags list_).

Can someone shed some light into this? Is my understanding of the process wrong? is **dag_file_processor_timeout** relevant for each individual dag or for processing all dags?

**Comment by toaryangupta** [comment]
hi @ErandM50 , were you able to find any answer or resource for your question.
